assets/concept_art/planet_generation.jpeg.import

```diff
@@ -0,0 +1,35 @@
+[remap]
```

Comment:
For uniformity I think this should be a .jpg file (we do have one .jpeg file in the game, but that's more of an oversight).